src/chainstate/burn/db/sortdb.rs

```diff
@@ -1794,6 +1794,106 @@ impl<'a> SortitionHandleConn<'a> {
 
         Ok(result)
     }
+
+    pub fn get_last_anchor_block_hashPSQ(&self) -> Result<Option<BlockHeaderHash>, db_error> {
```

Comment:
forgot about those, any reason why the original ones use a `&mut self` while it was not needed, which makes it impossible to call these without a mutable db ref.
So I added mines, obviously to be renamed, but I would think the mutable flavor should be removed, unless I'm missing something rather obvious